app/src/main/java/org/stepic/droid/notifications/badges/NotificationsBadgesListener.kt

```diff
@@ -0,0 +1,6 @@
+package org.stepic.droid.notifications.badges
+
+interface NotificationsBadgesListener {
+    fun hideBadge()
+    fun setBadge(count: Int)
```

Comment:
I expect that `Listener` should react on some events and methods should be named with `on` prefix. For example, `onBadgeCountChanged`